test/acceptance/create_webdriver.robot

```diff
@@ -31,6 +31,39 @@ Create Webdriver With Bad Keyword Argument Dictionary
     Run Keyword And Expect Error    kwargs must be a dictionary.
     ...    Create Webdriver    Firefox    kwargs={'spam': 'eggs'}
 
+Get Browser Capabilities Using Dictionary
+    [Documentation]    Gets current browser capabilities from WebDriver
+    [Setup]    Run Keywords    Cannot Run In RF Pre 2.9    Set Driver Variables
+    Create Webdriver    ${DRIVER_NAME}
+    ${capabilities}=    Get Browser Capabilities
+    Log    Capabilities = ${capabilities}
+    ${browser}=    Normalize Browser Name
+    Should Match    ${capabilities['browserName']}    ${browser}
```

Comment:
Change tests that it checks keys and does not care about the value. Because value changes between browser and perhaps Selenium version. 

Test in the unit test level that correct value is in correct key.